handlers.go

```diff
@@ -4,44 +4,102 @@ import (
 	"fmt"
 	"net/http"
 
+	"encoding/json"
 	"github.com/Financial-Times/go-fthealth/v1a"
-	"regexp"
+	log "github.com/Sirupsen/logrus"
```

Comment:
I could add it to the tech forum for this week seeing as there is only one other point to discuss?